src/routes/profile.js

```
@@ -86,6 +108,22 @@ router.post('/mod/ajax/processavatarrequest', isModMiddleware, (req, res) => {
       } else if (req.body.decision === false || req.body.decision === 'false') {
         console.log(`search lower user: ${foundReq.forUsername.toLowerCase()}`);
 
+        const pattern = /pending_avatars\/.*$/;
+
+        await s3.rejectAvatarRefactorFilePath(
+          foundReq.resLink.match(pattern)[0],
```

What happens if there's no match? Let's throw an error.